index.html

```diff
@@ -0,0 +1,738 @@
+        </p>
+      </div>
+    </article>
+  </div>
+  <div class="love-wrapper">
+    <section class="latest-news">
+      <h2 class="ln-heading">Latest news <i class="material-icons">keyboard_arrow_right</i></h2>
+      <div class="ln-mobile-atcl-wrapper">
+        <a href="#" class="ln-bg-img img1"></a>
+        <div class="ln-atcl-head">
+          <h3>Can face masks really protect us from toxic pullutants?</h3>
+          <p>The Conversations<span> - </span>4 hours ago</p>
+        </div>
+      </div>
+      <div class="ln-mobile-atcl-wrapper">
+        <a href="#" class="ln-bg-img img2"></a>
+        <div class="ln-atcl-head">
+          <h3>Norway pushes to electrify all domestic flights by 2040</h3>
+          <p>Matthew Beedham <span> - </span>5 hours ago</p>
+        </div>
+      </div>
+      <div class="ln-mobile-atcl-wrapper">
+        <a href="#" class="ln-bg-img img3"></a>
+        <div class="ln-atcl-head">
+          <h3>Scientists set up new telescopes dedicated to finding aliens</h3>
+          <p>Ivan Mehta <span> - </span>6 hours ago</p>
+        </div>
+      </div>
+      <div class="ln-mobile-atcl-wrapper">
+        <a href="#" class="ln-bg-img img4"></a>
+        <div class="ln-atcl-head">
+          <h3>Satoshi Nakaboto: ‘Bitcoin plunges alongside global markets’</h3>
+          <p>Satoshi Nakaboto <span> - </span>8 hours ago</p>
+        </div>
+      </div>
+      <div class="ln-mobile-atcl-wrapper">
+        <a href="#" class="ln-bg-img img5"></a>
+        <div class="ln-atcl-head">
+          <h3>Almost half of Indian startups don’t hire women to save on maternity costs</h3>
+          <p>Cara Curtis <span> - </span>6 hours ago</p>
+        </div>
+      </div>
+      <div class="ln-mobile-atcl-wrapper">
+        <a href="#" class="ln-bg-img img6"></a>
+        <div class="ln-atcl-head">
+          <h3>Here's your last chance to win a free iPhone 11, AirPods and charging pad</h3>
+          <p>TNW Deals <span> - </span>7 hours ago</p>
+        </div>
+      </div>
+      <div class="ln-mobile-atcl-wrapper">
+        <a href="#" class="ln-bg-img img7"></a>
+        <div class="ln-atcl-head">
+          <h3>John Oliver roasts Disney-owned Hotstar for making cuts to his show in India</h3>
+          <p>Ivan Mehta <span> - </span>8 hours ago</p>
+        </div>
+      </div>
+      <div class="ln-mobile-atcl-wrapper">
+        <a href="#" class="ln-bg-img img8"></a>
+        <div class="ln-atcl-head">
+          <h3>How Slack keeps employees focused while being a distracting app</h3>
+          <p>Nir Eyal <span> - </span>8 hours ago</p>
+        </div>
+      </div>
+
+    </section>
+    <section class="latest-funding">
+      <h2>Latest Funding rounds on index.co ></h2>
+      <ul class="lf-links">
+        <li>See More<span class="lf-links-bars"> | </span></li>
+        <li>Get Updates by email<span class="lf-links-bars"> | </span></li>
+        <li>Add Info</li>
+      </ul>
```

Comment:
This section is not behaving exactly as the original one in the medium breakpoint. Check the images below, it's just a minor change in the way the subtitles are arranged. Yours look like this:
![Screenshot from 2020-03-13 21-15-17](https://user-images.githubusercontent.com/51926937/76673945-ebcbd200-656f-11ea-9f02-105e4c851a1f.png)

The original is:
![Screenshot from 2020-03-13 21-15-23](https://user-images.githubusercontent.com/51926937/76673953-f7b79400-656f-11ea-9d74-e21627644f4f.png)